verilog/tools/kythe/verilog_kythe_extractor.cc

```diff
@@ -73,42 +63,43 @@ static void ExtractKytheFacts(
     if (!f.good()) {
       LOG(FATAL) << "Can't write to " << output_path;
     }
-    f << verilog::kythe::KytheFactsPrinter(facts_tree) << std::endl;
+    f << verilog::kythe::KytheFactsPrinter(file_list_facts_tree) << std::endl;
   }
+
+  return exit_status;
 }
 
 int main(int argc, char** argv) {
   const auto usage =
       absl::StrCat("usage: ", argv[0], " [options] <file> [<file>...]\n",
                    R"(
 verilog_kythe_extractor is a simple command-line utility
-to extract kythe indexing facts from the given file.
+to extract kythe indexing facts from the given file list.
 
-Expected Input: verilog file.
+Expected Input: verilog file list which contains the path of verilog files.
```

Comment:
Clarify that these files are top-level translation units, and should exclude `` `include``-d files.